neo/SmartContract/Native/ContractMethodAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Neo.SmartContract.Native
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    internal class ContractMethodAttribute : Attribute
```

Comment:
`NativeContractMethodAttribute`?

Reply:
I thought so at the beginning. But I find that it's under `Native` namespace.